src/components/clusters/ClustersListToolbar.tsx

```diff
@@ -114,26 +140,29 @@ const ClustersFilterToolbar: React.FC<ClustersFilterToolbarProps> = ({
             ))}
           </Select>
         </ToolbarFilter>
-
-        <ToolbarItem>
-          <InputGroup>
-            <TextInput
-              name="search-name"
-              id="search-name"
-              type="search"
-              aria-label="cluster name to be searched"
-              onChange={onSearchNameChanged}
-              value={searchName}
-              placeholder="Filter by name ..."
-            />
-            <Button variant={ButtonVariant.control} aria-label="search cluster name button">
-              <SearchIcon />
-            </Button>
-          </InputGroup>
-        </ToolbarItem>
+        <ToolbarButton
+          variant={ButtonVariant.primary}
+          onClick={() => history.push(`${routeBasePath}/clusters/~new`)}
+          id="button-create-new-cluster"
+          data-ouia-id="button-create-new-cluster"
+        >
+          Create New Cluster
```

Comment:
```suggestion
          Create cluster
```

I know we've had this button for a while, but `New` is somewhat redundant and sentence case `cluster` aligns with OCM's button.

Reply:
Done